client/src/components/SignUp/index.tsx

```diff
@@ -18,6 +19,21 @@ const SignUp = () => {
   const [isInfoEntered, setIsInfoEntered] = useState(false);
   const [isSignupComplete, setIsSignupComplete] = useState(false);
 
+  const onSignUp = async () => {
+    const watIAMUserId = email.split('@')[0];
+    await signup({
+      firstName,
+      lastName,
+      studentNumber: +studentNumber,
```

Comment:
What does this mean? Looks cool

Reply:
hahaha it's to convert `studentNumber` from `string` to `number`.

Also `studentNumber` is must be a valid number string since we use `type="number"` on its `TextField`. Otherwise we could run into studentNumber being converted to `NaN` here if it isn't a valid number.